kore/src/Kore/Step/Step.hs

```diff
@@ -400,9 +401,12 @@ finalizeRulesParallel
 finalizeRulesParallel initial unifiedRules = do
     let initialCondition = Conditional.withoutTerm initial
     results <-
-        Foldable.fold <$> traverse (finalizeRule initialCondition) unifiedRules
-    let unifications = MultiOr.make (Conditional.withoutTerm <$> unifiedRules)
-        remainder = Predicate.fromPredicate (Remainder.remainder' unifications)
+        Foldable.fold
```

Comment:
Why did this change?

Reply:
Honestly, I'm not sure. It wasn't exceeding 80 characters. I'll change it back